lib/beaker/host_prebuilt_steps.rb

```diff
@@ -107,6 +107,7 @@ def validate_host host, opts
         when host['platform'] =~ /cumulus/
           check_and_install_packages_if_needed(host, CUMULUS_PACKAGES)
         when (host['platform'] =~ /windows/ and host.is_cygwin?)
```

Comment:
It seems a little odd to have both `host.is_cygwin?` *and* `host.cygwin_installed?` defined (with a possibility of having `is_cygwin?` be true, but `cygwin_installed?` returning `false`). Do the docs clearly explain the difference?